ansible/roles/shared_dir/tasks/setup_host.yml

```diff
@@ -12,6 +12,20 @@
     - nfs
     - shared_files
 
+- name: Format system data disk
+  sudo: yes
+  filesystem: fstype=ext4 dev={{ datadisk_device }}1
```

Comment:
why does the `1` get added here? i've seen it before but wasn't sure why it was hardcoded here instead of the device name